composer.json

```diff
@@ -15,7 +15,8 @@
     "wp-cli/wp-cli-bundle": "^2.2",
     "wp-coding-standards/wpcs": "^2.2",
     "wp-phpunit/wp-phpunit": "^5.4",
-    "wpsh/local": "^0.2.3"
+    "wpsh/local": "^0.2.3",
+    "wp-premium/gravityforms": "^2.4.17"
```

Comment:
@kidunot89 we need to find another way to pull in this plugin and test. While GPL code on in `wp-premium` Org is technically legal (I'm not a lawyer 🤐), we will not support it as it takes away from the developers who built these plugins and poses as a potential security risk.

Reply:
Maybe we can pull it in as a zip someway? I have a developer's license, so access is not a problem.

Reply:
I'm open to ideas, it's there a recommend method you have for pull this plugin in?

Reply:
I am also open to ideas. One that comes to mind is we have a private S3bucket (or other storage service) from which we can read with private keys in Github. The we can use an archive installer with Composer to pull in those packages. 

I checked, but did not find any way to add files to Github or Travis for pulling in the data, but the above solution may be a good start.

@kasparsd any ideas?

Reply:
Honestly, I'm not sure how to handle this. How about we skip the tests if the dependency is not found for now?